README.md

```diff
@@ -22,7 +22,7 @@ More specifics to notice about the target:
 - The sign in page is located at `/user_sign_in`.
```

Comment:
I'm wondering if we could/whether we should lift this restriction on implementation details, and instead visit the homepage and click "Sign up" / "Sign in" instead.

This also makes me wonder whether we should add a test to ensure that "Sign in" / "Sign out" / etc are correctly conditionally displayed. We can defer this until later, however, as I want to publish the assignment as soon as possible this morning.